src/equations/shallow_water_multilayer_1d.jl

```diff
@@ -316,7 +315,7 @@ When the bottom topography is nonzero this scheme will be well-balanced when use
 nonconservative [`flux_nonconservative_ersing_etal`](@ref).
 
 To obtain an entropy stable formulation the `surface_flux` can be set as
-[`FluxPlusDissipation(flux_ersing_etal, DissipationLocalLaxFriedrichs()), flux_nonconservative_ersing_etal`](@ref).
```

Comment:
Did this ref get removed because there are versions of, e.g., `flux_ersing_etal` in Trixi and TrixiSW?

Reply:
I did not manage to resolve the links correctly with the function arguments, so I removed the reference.

Reply:
That makes sense, the `@ref` probably gets confused with which function it should jump to between the four available within the tick marks.